### ISSUE TITLE
webui-file-new-access task failure causes JSON serialization error

### ISSUE THREAD

**Issue by gjost** [densho/ddr-local#238]
User uploaded a master file to densho-26-144.
The upload succeeded, but there was no thumbnail.
`download full size` worked, but `download access` did nothing.
No access file for the binary in in `var/www/media...`.
User clicked on `generate new access file`, after which browser landed on the `collections-list` page but with the `TypeError`.

It looks like Celery is not able to serialize or deserialize an action record because the dict contains an `Identifier` object rather than an `id` string.
```
TypeError at /ui/collections/
<webui.identifier.Identifier file:ddr-densho-26-144-master-705b3ae653> is not JSON serializable
```

`request.session.sav
[TypeError-at-ui_collections.html.tar.gz](https://github.com/densho/ddr-local/files/2182319/TypeError-at-ui_collections.html.tar.gz)
e()` is trying to save the result of the `webui-file-new-access` action to `celery-tasks`.  The result is the dict of a `File` object.  `File.identifier` is an actual `Identifier` object, which Celery is unable to serialize to Redis.

```
{"celery-tasks": {
    "1ad2fff0-a9ee-443c-b5f7-5bb0a58d8aeb": {
        "action": "webui-file-new-access",
        "dismissable": True,
        "entity_id": "ddr-densho-26-144",
        "entity_url": "/ui/entity/ddr-densho-26-144/",
        "file_url": "/ui/file/ddr-densho-26-144-master-705b3ae653/",
        "filename": "ddr-densho-26-144-master-705b3ae653.tif",
        "message": "Generated new access file for <a href="/ui/file/ddr-densho-26-144-master-705b3ae653/">ddr-densho-26-144-master-705b3ae653.tif</a> (<a href="/ui/entity/ddr-densho-26-144/">ddr-densho-26-144</a>).",
        "result": {
            "access_abs": "/var/www/media/ddr/ddr-densho-26/files/ddr-densho-26-144/files/ddr-densho-26-144-master-705b3ae653-a.jpg",
            "access_rel": "files/ddr-densho-26-144/files/ddr-densho-26-144-master-705b3ae653-a.jpg",
            "basename": "ddr-densho-26-144-master-705b3ae653.tif",
            "basename_orig": "ddr-densho-26-144_master.tif",
            "collection_id": "ddr-densho-26",
            "collection_path": "/var/www/media/ddr/ddr-densho-26",
            "digitize_person": "",
            "entity_files_path": "/var/www/media/ddr/ddr-densho-26/files/ddr-densho-26-144/files",
            "entity_id": "ddr-densho-26-144",
            "entity_path": "/var/www/media/ddr/ddr-densho-26/files/ddr-densho-26-144",
            "ext": ".tif",
            "external": None,
            "external_urls": "",
            "id": "ddr-densho-26-144-master-705b3ae653",
            "identifier": <webui.identifier.Identifier file:ddr-densho-26-144-master-705b3ae653>,
            "idparts": ["ddr",
                        "densho",
                        26,
                        144,
                        "master",
                        "705b3ae653"],
            "json_path": "/var/www/media/ddr/ddr-densho-26/files/ddr-densho-26-144/files/ddr-densho-26-144-master-705b3ae653.json",
            "json_path_rel": "files/ddr-densho-26-144/files/ddr-densho-26-144-master-705b3ae653.json",
            "label": "Front",
            "links": None,
            "md5": "57a9c383da50f8babd81d4d2bf214511",
            "mimetype": "image/tiff",
... <trimmed 7519 bytes string>
```

Version info
```
loc: 51bd7dbd3e99e8d23e85092ce9b10f61f4bfb427  (HEAD -> master, origin/master) 2018-06-29 11:54:41 -0700
cmd: 7d27c206fe7e0905d6d772305b7b9c95cc027e4e  (HEAD -> master, origin/master, origin/HEAD) 2018-06-29 11:51:37 -0700
def: 4d83d8c372f9fbb011be4b21412d625533eea28a  (HEAD -> master, origin/master, origin/HEAD) 2018-06-20 20:10:06 -0700
def: /opt/ddr-local/ddr-defs/'
```

**Comment by gjost** [comment]
Django error page
[TypeError-at-ui_collections.html.tar.gz](https://github.com/densho/ddr-local/files/2182328/TypeError-at-ui_collections.html.tar.gz)

**Comment by pkikawa** [comment]
issue resolved. confirmed working as of v2.7.1c

**Comment by gjost** [comment]
Fixed again in ddr-local commit c7315c7 and ddr-cmdln commit 786558b for package ddrlocal-master_5.0.4~deb10.
